src/schema/schema.graphql

```diff
@@ -3102,6 +3149,14 @@ type Query {
 		traverseMountpoints: Boolean
 		folder: GetFolderFolderInput
 	): Folder
+	getHAB(
+		habRootGroupId: ID
+	): HabGroup
+	getDistributionListMembers(
+		limit: Int,
```

Comment:
Do you want it to change to Float? 
This question is in the context of the issue we faced recently with our customer.

Reply:
I doubt we will pass limit which is very high

Reply:
Limit would never be that high, so keeping it to Int only. I did use Int by following Zimbra API Docs.